broker-cli/commands/health-check.js

```diff
@@ -1,3 +1,4 @@
+const LndEngine = require('lnd-engine')
```

Comment:
can you add a todo in here to make this reference a more generic implementation?